requirements.txt

```diff
@@ -2,11 +2,15 @@
 click==8.1.3
 coverage==7.2.3
 grablib==0.8
-pytest==6.2.5
+pytest==7.3.1
```

Comment:
This won't work, needs reverting (see the only other open PR).

Reply:
The reason I did that was because I got this error from pip: 

> ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
> pytest-asyncio 0.21.0 requires pytest>=7.0.0, but you have pytest 6.2.5 which is incompatible.

Reply:
... and when I install `pytest==6.2.5` instead of 7.3.1, I get this error:

> ImportError: cannot import name 'Config' from 'pytest' (...\code\aiohttp-devtools\venv\lib\site-packages\pytest\__init__.py)

Reply:
Last test run installed `0.20.3` only 2 weeks ago. Dependency resolver should install the correct version normally.

Reply:
Thanks, adding `pytest-asyncio==0.20.3` to requirements.txt fixes the issue for me. Will force push the update in a minute.

Reply:
I suspect it was just a problem with your environment. Maybe outdated pip not doing dependency resolution properly, or not installing packages in a fresh env, maybe..